### PR TITLE
OCLOMRS-930 - Created back arrow on the create pages(Dictionary, Source & Organistion) and General details pages

### PR DESCRIPTION
**Pull request by Irenyak1**
Issue worked on:
# JIRA TICKET NAME:
[OCLOMRS-930](https://issues.openmrs.org/browse/OCLOMRS-930)

# Summary:
- I Created back arrow on the create pages(Dictionary, Source & Organistion) and General details pages
- The tests are all running.

### CODE REVIEW

**Comment by HerbertYiga** [comment]
thanks @Irenyak1,you can add a ticek id to your commit too @hadijahkyampeire ,how does this look to you

**Comment by Irenyak1** [comment]
Alright @HerbertYiga  I will be doing that. Thanks.

File: src/App.tsx [hunk 1]
```diff
@@ -9,7 +9,8 @@ import { AuthenticationRequired, LoginPage } from "./apps/authentication";
 import { Provider } from "react-redux";
 import store from "./redux";
 import { InProgressPage } from "./apps/notifications";
-import { Header, NavDrawer } from "./components";
+import { NavDrawer } from "./components";
+// import { Header, NavDrawer } from "./components";
```
Comment (hadijahkyampeire) [inline src/App.tsx:13]: minor: remove this comment

File: src/apps/dictionaries/utils.ts [hunk 2]
```diff
@@ -32,3 +32,17 @@ export const dictionaryNameFromUrl = (url: string): string => {
   let words = url.split("/");
   return words[words.length - 2];
 };
+
+export const getDictionaryTypeFromPreviousPath = (previousPath: String) => {
```
Comment (hadijahkyampeire) [inline src/apps/dictionaries/utils.ts:36]: Nice one!

**Review by hadijahkyampeire — APPROVED** [review]
LGTM, nice work @Irenyak1 , just remove the comment.

File: src/apps/dictionaries/utils.ts (hunk 2 above)
Comment (Irenyak1) [reply src/apps/dictionaries/utils.ts:36]: Thanks @hadijahkyampeire. Let me delete the comment and push

**Comment by Irenyak1** [comment]
@hadijahkyampeire  am wondering why travis is failing.

**Comment by hadijahkyampeire** [comment]
Let me check it and fix it.

**Comment by HerbertYiga** [comment]
> @hadijahkyampeire am wondering why travis is failing.

looking at the logs cypress is throwing multiple errors for example  CypressError: `cy.click()` failed because it requires a DOM element. could you try getting to the travis logs and see the breaking points

**Comment by Irenyak1** [comment]
@hadijahkyampeire  let me just delve into cypress and fix that.

**Pull request merged**